product-import/src/constants/index.ts

```diff
@@ -0,0 +1,13 @@
+export const APPLICATION_VERSION = 'v1';
+
+export const APPLICATION_REGION = process.env.AWS_REGION;
```

Comment:
cool, didn't know that `process.env` already contains info about region and there is no need to pass it manually :)